@xen-orchestra/lite/src/components/infra/InfraHostItem.vue

```diff
@@ -18,6 +18,9 @@
           @click="toggle()"
         />
       </template>
+      <li class="vm-number" v-if="!isExpanded && faAngleUp">
+        {{ getVmCount }} VMs
```

Comment:
To match the figma visual : `{{ ... }} VMs running`.
Also use i18n. See `en.json` and `fr.json`